src/Integration.Vsix/CFamily/PortedFromJava/MsvcDriver.cs

```diff
@@ -436,9 +426,17 @@ public static Request ToRequest(CFamilyHelper.Capture[] captures)
                   // TODO can be passed directly to Clang as PreprocessorOptions.Includes
                   + includes.ToString();
                 request.TargetTriple = probe.Isx64 ? "x86_64-pc-windows" : "i686-pc-windows";
-                if (files.Count == 1)
+
+             
+                switch (files.Count)
```

Comment:
I think it makes sense to move this part to the end of the switch. As `files` isn't modified after the switch => avoid executing all the previous code for nothing

Reply:
Not sure I follow - can you clarify please?

Reply:
Checking the `files` count is now happening at the end of the function `ToRequest` for no real reason. To fail early you can move it just after the switch (L271)